wordpress-mobile/WordPress-iOS
# My Sites: new view when all sites are hidden

Fixes #8486

**To test:**
- Hide all sites on an account.
- Verify view appears as:
![all_sites_hidden](https://user-images.githubusercontent.com/1816888/35295581-ac1dd00e-0036-11e8-8cb3-3b7decc8240d.png)
